backend/src/main/java/ai/verta/modeldb/artifactStore/storageservice/s3/S3Client.java

```diff
@@ -112,7 +112,17 @@ private void initializeWithTemporaryCredentials(Regions awsRegion)
     AWSSecurityTokenService stsClient = null;
     AmazonS3 newS3Client = null;
     try {
-      stsClient = AWSSecurityTokenServiceClientBuilder.standard().withRegion(awsRegion).build();
```

Comment:
This is code from https://github.com/VertaAI/modeldb/pull/2868 that should be merged now...not sure why it's showing up as a change here.